ComfortZoneApp/app/src/main/java/com/example/comfortzone/Fragments/FlightFragment.java

```diff
@@ -23,6 +23,7 @@
 import com.google.gson.Gson;
 
 import java.util.List;
+import java.util.concurrent.TimeUnit;
 
 public class FlightFragment extends Fragment {
```

Comment:
remember fragment/activities should only think about view as much as possible, think about how we can group related methods into different helper classes to help delegate the business logic